### DISCUSSION TITLE
BindingResolutionException on Laravel 8

### DISCUSSION THREAD

**Discussion by KFoobar** [spatie/laravel-responsecache#329] [Q&A] [answered]
Tried to install this package on a fairly fresh Laravel 8.18.1 application. Followed the steps under the installation section of the documentation. But it results in BindingResolutionException.

```
Illuminate\Contracts\Container\BindingResolutionException
Target class [] does not exist.
```

I can trace it to:

```
$this->app->bind(Serializer::class, function (Container $app) {
    return $app->make(config('responsecache.serializer'));
});
```

Have is missed something?

**Comment by jonesheckler** [comment]
Hi

I have the same problem, any tips on how to solve it?

**Comment by jonesheckler** [comment, marked as answer]
I found the problem here.

It was just laravel cache.

:)

**Comment by IgorRubas** [comment]
Hi all, in my case (installed on a macbook docker Laravel8), the solution was to add 1 line to the routers/web.php file - <?php
namespace App\Http\Controllers; and this issue was closed. But I have a strong feeling that this is not the last issue. Before the solution there was a dance with sabers. 

Solution:
routers/web.php file - <?php
namespace App\Http\Controllers;
